invenio_files_rest/models.py

```diff
@@ -141,6 +141,32 @@ def inner(self, *args, **kwargs):
     return decorator
 
 
+class BucketError(object):
```

Comment:
Should they not inherit from ``Exception``?

Reply:
These two classes are patterned over https://github.com/inveniosoftware/invenio-rest/blob/d5a3d185c0a8c8cd46860bd0fd9be8bb3786f157/invenio_rest/errors.py#L35-L58, which also doesn't inherit from `Exception`, because it's not intended to be thrown, but is just representing an error.